index.html

```diff
@@ -189,16 +199,37 @@ <h2>Languages</h2>
                     </div>
                   </div>
                   <div class="Frame flex-row">
-                    <p>Frameworks</p>
+                    <h1 class="framework">Frameworks</h1>
                     <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
                   </div>
                   <div class="skill flex-row">
-                    <p>Skills</p>
+                    <h4>Skills</h4>
                     <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
                   </div>
                 </div>
               </section>
         </section>
+
+        <section>
+          <div class="contact-form">
+            <div class="contact-writeup">
+              <h1>Contact me</h1>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                 I'd love to help with it</p>
+            </div>
+
+            <form>
+              <div class="contact-form-div">
+                <input type="text" placeholder="enter name" class="contact-input" maxlength="30" required>
+                <input type="text" placeholder="enter email" class="contact-input" required>
+                <textarea name="" id="" cols="45" rows="10" class="contact-text" placeholder="write your message here"
+                name="message" maxlength="500" required
+                ></textarea>
+                <button>Get in touch</button>
+              </div>
+            </form>
```

Comment:
- Currently your form doesn't have a `method` nor an `action` and as such it cannot send data, it is important that users of your portfolio can reach out to you to request for your services and the one way to do this is by making your form functional enough to receive this information and you can do this using the example below however you will need the information to be sent somewhere, to achieve this microverse recommends using [formspree service](https://formspree.io/html). please make sure to follow the step to setup your formspree account and attach to where you have action in the example below :+1:
```html
<form method="POST" action="https://formspree.io/f/{form_id}">

</form>
```
- Note, where you have a form id in the example you are to put in your formspree id.